extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -357,11 +367,20 @@ const lastStatusDescription = computed(() => {
 });
 
 const toolTipText = computed(() => {
+  let entrypoint = "unknown";
+  if (
+    home.selectedConfiguration &&
+    !isConfigurationError(home.selectedConfiguration) &&
+    home.selectedConfiguration.configuration.entrypoint
+  ) {
+    entrypoint = home.selectedConfiguration.configuration.entrypoint;
+  }
```

Comment:
Same here, a new store getter could help on keeping this component's responsibilities a bit cleaner.
```js
const toolTipText = computed(() => {
  const entrypoint = home.currentEntrypoint || "unknown";
  
  return `Deployment Details
...
```

Reply:
The tooltip text feels appropriate in the component to me since it is the only use of it. Not opposed to having it in the store, but I would look for it in the component first instinctually

Reply:
If we were sharing the value, I could justify it being in the store, but right now, it feels better to me in the component as well.